drivers/sensor/bosch/bma4xx/bma4xx.c

```diff
@@ -202,7 +202,7 @@ static int bma4xx_attr_set_bwp(const struct device *dev, const struct sensor_val
 /**
  * @brief Implement the sensor API attribute set method.
  */
-static int bma4xx_attr_set(const struct device *dev, enum sensor_channel chan,
+static int bma4xx_attr_set(const struct device *dev, enum sensor_channel ch,
```

Comment:
Same here

Reply:
attr set and attr get are not covered by this PR as that would be a breaking API change, but if the name of the variable needs changing I'll take a closer look at. thank you for spotting these things!

Reply:
Agreed, rename should not have occurred, undone